src/config/Cargo.toml

```diff
@@ -15,4 +15,6 @@ appveyor = { repository = "amethyst/amethyst" }
 travis-ci = { repository = "amethyst/amethyst" }
```

Comment:
Before we push out the new `amethyst_config` release, please replace this:
```
appveyor = { repository = "amethyst/amethyst" }
```
with this:
```
appveyor = { repository = "amethyst/amethyst", branch = "develop" }
```